pcb/rules/F6_2.py

```diff
@@ -15,28 +16,42 @@ def check(self):
         """
         Proceeds the checking of the rule.
         The following variables will be accessible after checking:
-            * pads_bounds
-            * pads_distance
-            * right_anchor
+            * center_pads
+            * center_fab
         """
         module = self.module
         if module.attribute != 'smd':
             # Ignore non-smd parts
             return False
 
-        center = module.padMiddlePosition()
+        center_pads = module.padMiddlePosition()
+        center_fab  = module.geometricBoundingBox("F.Fab").center
+
+        if center_pads['x'] != center_fab['x'] or center_pads['y'] != center_fab['y']:
+            self.warning("Footprint centers of pads and F.Fab do not match")
```

Comment:
I would not call this a warning. A lot of footprints would result in this and it would just confuse users.

Reply:
How about moving the warning output into the code-path that displays the error (if there is any). Then the user has the additional info on which two origins were checked.
Otherwise it will be a mystery for the user why exactly the check failed.

Reply:
Well if the check fails then i would assume your script knows why and can generate a clear output (you could for example word it such that it is clear which options you checked). This warning on its own however might be read as "i need to ensure the centers are equal" which is not the case.

Reply:
I spend some time on better wording, but could not come up with something good. 

Actually I don't know why I introduced it in the first place. There is no real use-case for it.
Now one line is shown for each calculated center. This has all the information the user needs. Its also much clearer than the previous version that had all coordinates in one line.

New look:
![grafik](https://user-images.githubusercontent.com/20518746/84761319-cd370600-afb8-11ea-976d-5113ff940640.png)